suite-native/theme/src/useGetSystemColorVariant.ts

```diff
@@ -0,0 +1,11 @@
+import { useColorScheme } from 'react-native';
+
+import { ThemeColorVariant } from '@trezor/theme';
+
+export const useGetSystemColorVariant = (): ThemeColorVariant => {
+    const colorScheme = useColorScheme();
+    if (colorScheme === 'dark') {
+        return 'dark';
+    }
+    return 'standard';
+};
```

Comment:
Why is this necessary? Basically it enforces only `dark` / `standard` theme in inverted theme provider?

Reply:
Seems quite redundent at this moment